### PR TITLE
docs: Robotoff

### PR DESCRIPTION
**Pull request by teolemon**
### What
docs: Robotoff

### CODE REVIEW

File: docs/reference/api-tutorials/robotoff.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+About
+The Robotoff project is intended to complete missing information of products by prompting users to confirm predictions inferred by Artificial Intelligence algorithms. These algorithms are calculated based on "insights", which are facts about a product that have been extracted or deduced from the product pictures, ingredients, categories, labels, etc... 
+The OpenAPI documentation of the Robotoff API is located at : https://openfoodfacts.github.io/robotoff/references/api/#tag/Questions/operation/getQuestionsRandom 
+
+The more general Robotoff documentation is located at : 
+https://openfoodfacts.github.io/robotoff/
```
Comment (alexgarel) [inline docs/reference/api-tutorials/robotoff.md:6]: I think this is more part of /intro/, don't you think so ?

Comment (teolemon) [reply docs/reference/api-tutorials/robotoff.md]: moved @alexgarel

Comment (teolemon) [reply docs/reference/api-tutorials/robotoff.md]: ok to merge ?

File: docs/introduction/robotoff.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+### About
+The Robotoff project is intended to complete missing information of products by prompting users to confirm predictions inferred by Artificial Intelligence algorithms. These algorithms are calculated based on "insights", which are facts about a product that have been extracted or deduced from the product pictures, ingredients, categories, labels, etc...
+
+### Robotoff OpenAPI documentation
+The OpenAPI documentation of the Robotoff API is located at : https://openfoodfacts.github.io/robotoff/references/api/#tag/Questions/operation/getQuestionsRandom 
+
+### Robotoff documentation
+The more general Robotoff documentation is located at : 
+https://openfoodfacts.github.io/robotoff/
```
Comment (VaiTon) [inline docs/introduction/robotoff.md:9]:
```suggestion
### Robotoff documentation
The documentation for Robotoff is located here:

- [General Documentation][general_docs]
- [API Documentation (OpenApi)][api_docs]

[general_docs]: https://openfoodfacts.github.io/robotoff/ "General Documentation"
[api_docs]: https://openfoodfacts.github.io/robotoff/references/api/ "API Documentation"
```

**Review by alexgarel — APPROVED** [review]
:+1:

(committed @VaiTon suggested change)

**Pull request merged**